data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -54,53 +56,124 @@
         {FLAG_VARIANT wood-elvish}
     [/side]
 
-    # rather than have the orcs + trolls be mushed together as generic baddies, let's have the trolls acting as guardians of the volcano and the orcs are more transient
+    # work-around until better AI control is found
```

Comment:
Please have the comment say what the issue that's being worked around is.

Reply:
added more to the comments